app/views/layouts/application.html.erb

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>MicroTutorials</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+<title>MicroTutorials</title>
+	<% flash.each do |key, value| %>
```

Comment:
Wyrzuć to do partiala shared/partials/_flash.haml
i wyrenderuj tutaj tylko partial